### ISSUE TITLE
Rule S3459  False positive.

### ISSUE THREAD

**Issue by gkardava** [SonarSource/sonar-csharp#1336]
### Description

Rule S3459 does not recognize assignment of a variable.   

### Repro steps
```C#
public class A {
        private readonly int _b; // Unassigned members should be removed
        public A(Func<(int, int)> init)
        {
            int a;
            (a, _b) = init();
        }
        public int GetB()
        {
            return _b;
        }
    }
```

* SonarC# Version  3.10 
* Visual Studio Version 17

**Comment by valhristov** [comment]
Thanks for reporting this problem. We already know about it and we will address it in the future.

Duplicate of #1153
